Source/WebCore/rendering/RenderGrid.cpp

```diff
@@ -1559,6 +1592,9 @@ void RenderGrid::populateGridPositionsForDirection(GridTrackSizingDirection dire
             positions[i + 1] = positions[i] + offset.distributionOffset + tracks[i].unclampedBaseSize() + gap;
         positions[lastLine] = positions[nextToLastLine] + tracks[nextToLastLine].unclampedBaseSize();
 
+        if (isMasonry(direction))
```

Comment:
Could you explain what this is for in the commit message? Not obvious how it's related to the changes we are making. How did this change contribute to us passing the aforementioned tests?